test/models/asyncapi_test.js

```diff
@@ -56,6 +56,29 @@ describe('AsyncAPIDocument', function() {
       expect(d.servers().test2.constructor.name).to.equal('Server');
       expect(d.servers().test2.json()).to.equal(doc.servers.test2);
     });
+
+    it('should return an empty object if the AsyncAPI document has not servers', function() {
```

Comment:
Does this not make more sense as title for the test or is it the same? :thinking:  
```suggestion
    it('should return an empty object if the AsyncAPI document has no defined servers', function() {
```

Reply:
Applied :)